src/frontend/src/lib/enums/progress-steps.ts

```diff
@@ -53,3 +53,9 @@ export enum ProgressStepsUpdateBalanceCkBtc {
 	RELOAD = 'reload',
 	DONE = 'done'
 }
+
+export enum ProgressStepsSendStepName {
```

Comment:
Yeah that's the thing, this is not a "Progress" step. This is a step of the "Wizard".

So I suggest to move it to a `wizard-steps.ts` module and name it `WizardStepsSend`.